Hangman/code.py

```diff
@@ -1,5 +1,17 @@
 import random  # import random library
 
+# method to draw hangman
+def draw_hangman(i):
+    Graphics =['''_____\n|/  |\n|   O\n|  /|\\\n|  / \\\n|''',
+    '''_____\n|/\n|   O\n|  /|\\\n|  / \\\n|''',
+    '''_____\n|/\n|   O\n|  /|\\\n|  /\n|''',
+    '''_____\n|/\n|   O\n|  /|\\\n|\n|''',
+    '''_____\n|/\n|   O\n|  /|\n|\n|''',
```

Comment:
I didn't understand this part,

could you explain what this part does?

Reply:
It is a list of hangman figures: see this commit - https://github.com/prateekiiest/Code-Sleep-Python/pull/96/commits/bcd381c3d21723745d5579cc142da61e5b57ba5e

Codeclimate marked this as an issue because the function length was 45 lines (<=25 lines guideline for good code). So, I modified to make it within limit.

Reply:
sure got it